ui_framework/components/index.js

```diff
@@ -82,15 +82,20 @@ export {
   KuiMenuItem,
 } from './menu';
 
-export * from './modal';
-
 export {
-  KuiPager,
-  KuiPagerButtonGroup,
-} from './pager';
+  KuiTabs,
+  KuiTab
+} from './tabs';
 
 export {
   KuiToolBarSearchBox,
   KuiToolBar,
   KuiToolBarFooter,
 } from './tool_bar';
+
+export * from './modal';
+
+export {
+  KuiPager,
+  KuiPagerButtonGroup,
+} from './pager';
```

Comment:
Would you mind moving these exports back to lines 85 and 88? I did some reordering in `master` which is probably what caused this conflict. 😄